src/AppInstallerSharedLib/Registry.cpp

```diff
@@ -132,6 +132,20 @@ namespace AppInstaller::Registry
 
             return true;
         }
+
+        bool TryDeleteRegistryValueData(const wil::shared_hkey& key, const std::wstring& valueName)

```

Comment:
This should probably be called `DeleteRegistryValueIfPresent`.  `Try` shouldn't throw, even on failure.